src/modules/urls/buildUrl.js

```diff
@@ -0,0 +1,17 @@
+import config from 'config'
+
+export async function formatUrlFromBuild(build, { absolute = false } = {}) {
```

Comment:
Shouldn't we make that module cross environment (browser/node)? I see some places where I could use it on the front.